tools/uncrustify.cfg

```diff
@@ -853,7 +853,7 @@ sp_extern_paren                 = ignore   # ignore/add/remove/force
 
 # Add or remove space after the opening of a C++ comment,
 # i.e. '// A' vs. '//A'.
-sp_cmt_cpp_start                = ignore   # ignore/add/remove/force
+sp_cmt_cpp_start                = add   # ignore/add/remove/force
```

Comment:
This should probably be "force" which means "make sure there is only one".

Reply:
True, that's an oversight of mine, I think I was testing if it made a difference and than forgot to put it back.